### DISCUSSION TITLE
How to recreate sumo projection

### DISCUSSION THREAD

**Discussion by benzom** [pyproj4/pyproj#1093] [Q&A]
After converting osm file to sumo network I got this info

```
<location netOffset="-395885.58,-6185860.67" convBoundary="0.00,0.00,3442.30,2845.95" origBoundary="37.338524,55.806366,37.393811,55.835368" projParameter="+proj=utm +zone=37 +ellps=WGS84 +datum=WGS84 +units=m +no_defs"/>
```

I would like to convert some (lat, long) coordinates to sumo projection coordinates. How can I do this? I know that some similar functions are provided by pyproj library, but don't understand, how to use it here

**Comment by snowman2** [comment]
This should be a helpful reference: https://pyproj4.github.io/pyproj/stable/examples.html#transformations-from-crs-to-crs
